include/filewriter/summarywriter_impl.hpp

```diff
@@ -5,7 +5,7 @@
 #ifndef MLBOARD_SUMMARY_WRITER_IMPL_HPP
 #define MLBOARD_SUMMARY_WRITER_IMPL_HPP
 
-#include <mlboard/core.hpp>
+#include "../core.hpp"
```

Comment:
I tried including using the following way it's working. I think it's because of `include_directories(${CMAKE_BINARY_DIR}/include)` snippet. Can you change the copy location to `build/include/mlboard/` instead of `build/include/` so that `#include <mlboard/core.hpp>` works?  


```
#include<core.hpp>
```

Reply:
Why do we need ``#include <mlboard/core.hpp>` , won't relative paths good ?

Reply:
Anyways I have made changed https://github.com/mlpack/mlboard/pull/5/commits/24414eb19317c4a2ac04919a5e0213baf66bcb44